src/config/web3/utils/index.ts

```diff
@@ -32,30 +32,32 @@ export const setupNetwork = async ({
     const { chainName, nativeCurrency, rpcUrls, blockExplorerUrls } = getChainData(network);
 
     try {
-      if (network === 1) {
-        await provider.request({
-          method: 'wallet_switchEthereumChain',
-          params: [
-            {
-              chainId,
-            },
-          ],
-        });
-        return true;
+      // @ts-ignore
```

Comment:
can we validate string type instead of ignoring?

Reply:
Thanks, I've fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/711/commits/375a50de5cbea756b2b03dfbc37a1df70c4cd5a1) commit.